modules/components/src/Arranger/QuickSearch/QuickSearchFieldsQuery.js

```diff
@@ -7,11 +7,11 @@ import { decorateFieldWithColumnsState } from './QuickSearchQuery';
 const nestedField = ({ field, nestedFields }) =>
   nestedFields.find(
     x =>
-      x.field ===
-      field.field
-        .split('.')
-        .slice(0, -1)
-        .join('.'),
+      x?.field ===
+      field?.field
+        ?.split('.')
+        ?.slice(0, -1)
```

Comment:
you don't need the `?` after `split`. `split` and `splice` never return null values